{
   "assignee" : null,
   "assignees" : [],
   "author_association" : "NONE",
   "body" : "Bitcoin Core version v0.18.99.0-4bfef0dae-dirty\r\nCompiled natively on \"Linux Debian-Desktop 4.4.185 #1 SMP Fri Jul 19 14:34:55 EDT 2019 aarch64 GNU/Linux\"\r\n\r\n```\r\n./autogen.sh\r\n./configure --disable-debug CFLAGS=\"-g0 -O3\" CXXFLAGS=\"-g0 -O3\" BDB_LIBS=\"-L${BDB_PREFIX}/lib -ldb_cxx-4.8\" BDB_CFLAGS=\"-I${BDB_PREFIX}/include\" --enable-werror --with-qrencode --enable-cxx --disable-shared --with-pic --with-boost-libdir=/usr/lib/arm-linux-gnueabihf\r\n```\r\nGUI runs OK. And noticeable faster than previously compiled (with debug enabled) v0.18.99.0-495db72ee-dirty. But both\r\n\r\n- Bitcoin Core version v0.18.99.0-495db72ee-dirty (debug enabled, default optimization)\r\n- Bitcoin Core version v0.18.99.0-4bfef0dae-dirty (debug disabled, -O3)\r\n\r\nare crashing when mouse cursor is pointed to File/OpenWallet menu item. Starting parameters are the same:\r\n\r\nbitcoin-qt -datadir=/bdb -nodebug -dbcache=1536\r\n\r\n/bdb is dedicated RAID1 volume with bitcoin database inside. Console output:\r\n\r\n```\r\nterminate called after throwing an instance of 'boost::filesystem::filesystem_error'\r\n  what():  boost::filesystem::status: Permission denied: \"/bdb/lost+found/wallet.dat\"\r\n```\r\nThere is nothing about the crash in debug.log before sudden application abort.\r\nI'm wondering for what purpose to access the wallets in lost+found folder? I thought there may be a problems with access rights for that folder. I have deleted lost+found and executed fsck -f /dev/md0 for /bdb directory volume. The problem still here.\r\nWhile configuring the build, I had to add\r\n```\r\n--with-boost-libdir=/usr/lib/arm-linux-gnueabihf\r\n```\r\nbecause configure script was unable to find boost executables.",
   "closed_at" : null,
   "closed_by" : null,
   "comments" : 0,
   "comments_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/16877/comments",
   "created_at" : "2019-09-15T21:27:16Z",
   "events_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/16877/events",
   "html_url" : "https://github.com/bitcoin/bitcoin/issues/16877",
   "id" : 493788363,
   "labels" : [
      {
         "color" : "FBBAAB",
         "default" : false,
         "id" : 64585,
         "name" : "Bug",
         "node_id" : "MDU6TGFiZWw2NDU4NQ==",
         "url" : "https://api.github.com/repos/bitcoin/bitcoin/labels/Bug"
      }
   ],
   "labels_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/16877/labels{/name}",
   "locked" : false,
   "milestone" : null,
   "node_id" : "MDU6SXNzdWU0OTM3ODgzNjM=",
   "number" : 16877,
   "repository_url" : "https://api.github.com/repos/bitcoin/bitcoin",
   "state" : "open",
   "title" : "armhf GUI: Permission denied: \"/database/lost+found/wallet.dat\" Aborted",
   "updated_at" : "2019-09-15T21:27:16Z",
   "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/16877",
   "user" : {
      "avatar_url" : "https://avatars2.githubusercontent.com/u/54221205?v=4",
      "events_url" : "https://api.github.com/users/Nikolay-Po/events{/privacy}",
      "followers_url" : "https://api.github.com/users/Nikolay-Po/followers",
      "following_url" : "https://api.github.com/users/Nikolay-Po/following{/other_user}",
      "gists_url" : "https://api.github.com/users/Nikolay-Po/gists{/gist_id}",
      "gravatar_id" : "",
      "html_url" : "https://github.com/Nikolay-Po",
      "id" : 54221205,
      "login" : "Nikolay-Po",
      "node_id" : "MDQ6VXNlcjU0MjIxMjA1",
      "organizations_url" : "https://api.github.com/users/Nikolay-Po/orgs",
      "received_events_url" : "https://api.github.com/users/Nikolay-Po/received_events",
      "repos_url" : "https://api.github.com/users/Nikolay-Po/repos",
      "site_admin" : false,
      "starred_url" : "https://api.github.com/users/Nikolay-Po/starred{/owner}{/repo}",
      "subscriptions_url" : "https://api.github.com/users/Nikolay-Po/subscriptions",
      "type" : "User",
      "url" : "https://api.github.com/users/Nikolay-Po"
   }
}
